### PR TITLE
Fixes loading of recipe content after clicking sidebar link

### LINKED ISSUE #101: Recipe sidebar links don't update content
Clicking on a recipe sidebar link updates the image, and panel header and footer, but not the main recipe content.

### PR DESCRIPTION
**Pull request by hermzz**
No need to make a AJAX call from the ingredients and directions list, the content is already included in the initial recipe request.

As a bonus also improves the rating star code.

Fixes #101

### CODE REVIEW

File: frontend/modules/recipe/components/Ratings.js [hunk 1]
```diff
@@ -4,19 +4,19 @@ require("./../css/ratings.scss");
 
 export default React.createClass({
   render: function() {
-    var tmp = '......';
-    var start_map = tmp.split(".", this.props.stars);
-    var empty_star_map = tmp.split(".", 5 - this.props.stars);
-    var stars = start_map.map(function(name, index){
+    const stars = this.props.stars ? this.props.stars : 0;
```
Comment (RyanNoelk) [inline frontend/modules/recipe/components/Ratings.js]: Thanks for cleaning up the horrible hack of how I process Stars. But right now the user can enter a value > 5 for stars.

We need to make sure that 5 is the max and 0 is the min:

```
    if (stars > 5) {
      stars = 5
    } else if (stars < 0) {
      stars = 0
    }
```

**Review by RyanNoelk — CHANGES_REQUESTED** [review]

File: frontend/modules/recipe/components/Ratings.js (hunk 1 above)
Comment (RyanNoelk) [reply frontend/modules/recipe/components/Ratings.js]: We can address the bigger problem of users entering large numbers in https://github.com/RyanNoelk/OpenEats/issues/79

**Comment by hermzz** [comment]
Good point, I feel this needs to be addressed in the API as well.

**Review by RyanNoelk — APPROVED** [review]
LGTM, I'll let u merge it still this will have conflicts with your other PR. Pick one that is easier for you to resolve.

**Pull request merged**